src/Microsoft.Extensions.Localization.Abstractions/LocalizedString.cs

```diff
@@ -64,6 +77,11 @@ public LocalizedString(string name, string value, bool resourceNotFound)
         public bool ResourceNotFound { get; }
 
         /// <summary>
+        /// The location which was searched for a locatization value.
+        /// </summary>
+        public string LocationSearched { get; }
```

Comment:
@hishamco - isn't that the behavior of the `ResourceManagerStringLocalizer` and not necessarily a trait of this type? Plus  the intent of making this change was to help diagnosing given that you have a localized string.